backends/tfhe-cuda-backend/src/bindings.rs

```diff
@@ -0,0 +1,1684 @@
+/* automatically generated by rust-bindgen 0.70.1 */
```

Comment:
What's the workflow with these bindings? Can't we generate them during compilation instead of adding this to the repository?

Reply:
It is generated at build time, but I have to commit the file to the repository otherwise cargo publish will complain when we release. That's why I introduced some logic to detect if headers have changed and regenerate the binding only then. There is a CI check that the most up to date bindings.rs file is committed to the repository.